app/src/main/java/org/commcare/dalvik/abha/ui/main/activity/TestActivity.kt

```diff
@@ -54,7 +54,7 @@ class TestActivity : AppCompatActivity() {
         val intent = Intent(action).apply {
             putExtras(
                 bundleOf(
-                    "mobile_number" to "",
+                    "mobile_number" to "9560833229",
```

Comment:
is this a real number ? If so, who does it belong to ?

Reply:
The TestActivity is for testing purpose. This activity basically used to launch and the test the all possible flows. We can say its acting as a proxy for CommCare APP.

The phone number belongs to me and its used to receive the OTP for authentication via ABDM api.